var/spack/repos/spack_repo/builtin/packages/grid/package.py

```diff
@@ -43,6 +43,38 @@ class Grid(AutotoolsPackage):
     )
     variant("timers", default=True, description="System dependent high-resolution timers")
     variant("chroma", default=False, description="Chroma regression tests")
+    variant("cuda", default=False, description="Build with CUDA support")
```

Comment:
Can we remove this one, given we inherit from `CudaPackage`:
```suggestion
```
?

Reply:
How would that work?  Does a package inheriting from `CudaPackage` automatically get a `cuda` variant?

Reply:
Yes. You can check `CudaPackage` to see which directives are inherited from the base class.